cloudwatchlogs_actions.go

```diff
@@ -7,34 +7,36 @@ package incite
 import (
 	"context"
 
-	"github.com/aws/aws-sdk-go/aws/request"
-	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
 )
 
 // CloudWatchLogsActions provides access to the CloudWatch Logs actions
 // which QueryManager needs in order to run Insights queries using the
 // CloudWatch Logs service.
 //
-// This interface is compatible with the AWS SDK for Go (v1)'s
-// cloudwatchlogsiface.CloudWatchLogsAPI interface and *cloudwatchlogs.CloudWatchLogs
-// type, so you may use either of these AWS SDK types to provide the
+// This interface is compatible with the AWS SDK for Go (v2)'s
+// *cloudwatchlogs.Client type, so you may use this AWS SDK type to provide the
 // CloudWatch Logs act capabilities.
```

Comment:
nit: (This typo was there before, but) we should change "act" to "action" on line 19